src/test/kotlin/uk/gov/justice/digital/hmpps/prisontonhs/services/health/HealthCheckIntegrationTest.kt

```diff
@@ -231,17 +229,6 @@ class HealthCheckIntegrationTest : IntegrationTest() {
   }
 
   private fun mockQueueWithoutRedrivePolicyAttributes() {
-    val queueName = ReflectionTestUtils.getField(queueHealth, "queueName") as String
-    val queueUrl = awsSqsClient.getQueueUrl(queueName)
-    whenever(
-      awsSqsClient.getQueueAttributes(
```

Comment:
not really understanding how this ever worked - for spies need to always call `doReturn` first otherwise the real one is called on the setup

Reply:
Ugh good find.  Looking forward to this code being rewritten when we move it into the queue lib.